test/conformance/testdata/conformance.txt

```diff
@@ -112,15 +112,15 @@ test/e2e/common/projected.go: "should be consumable from pods in volume as non-r
 test/e2e/common/projected.go: "should be consumable from pods in volume with mappings"
 test/e2e/common/projected.go: "should be consumable from pods in volume with mappings and Item Mode set"
 test/e2e/common/projected.go: "should be consumable in multiple volumes in a pod"
-test/e2e/common/projected.go: "optional updates should be reflected in volume"
+test/e2e/common/projected.go: "updates should be reflected in volume"
 test/e2e/common/projected.go: "should be consumable from pods in volume"
 test/e2e/common/projected.go: "should be consumable from pods in volume with defaultMode set"
 test/e2e/common/projected.go: "should be consumable from pods in volume as non-root"
 test/e2e/common/projected.go: "should be consumable from pods in volume with mappings"
```

Comment:
this one is still duplicated AFAICT (see line 112 currently)

Reply:
I made the full test case name unique, by adding in another Context().  However, the string inside the It() is still duplicated.

Do we need the It() strings to be unique?

Reply:
the other option I could do is to split out the tests into 3 different files: projected_secret, projected_configmap, projected_downwardapi

Reply:
The full identifier needs to be unique, as long as that's the case. I think you're right and that this will already work but it might be worth splitting up anyhow?